internal/pkg/aws/cloudformation/stackset/status_test.go

```diff
@@ -0,0 +1,251 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package stackset
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/stretchr/testify/require"
+)
+
+func TestOpStatus_IsCompleted(t *testing.T) {
```

Comment:
same here, all the `TestOpStatus_` tests are the same as before, I added the `TestInstanceStatus_` tests